### ISSUE TITLE
No such property: serviceProviders for class: java.lang.String

### ISSUE THREAD

**Issue by devacto** [DiUS/pact-jvm#254]
When I run `./gradlew build` I get the error `No such property: serviceProviders for class: java.lang.String` 

My `build.gradle` file contains the following:
```
pact {
    serviceProviders {
        banking_api {
            startProviderTask = 'startProvider'
            terminateProviderTask = 'stopProvider'
            hasPactWith('banking_api_consumer') {
                pactFile = file('pacts/banking_api_consumer-banking_api.json')
            }
        }
    }
}
```

And my `gradle.properties` file contains `pact = 3.1.0`.

To make it work, I had to delete `pact = 3.1.0` from my `gradle.properties` file.

**Comment by uglyog** [comment]
You will now get the following error: `Your project is misconfigured, was expecting a 'pact' configuration in the build, but got a String with value '3.1.0' instead. Make sure there is no property that is overriding 'pact'.`

**Comment by devacto** [comment]
Thanks Ron! You're a champ! :)

**Comment by uglyog** [comment]
Released fix with versions 3.2.8/2.4.10
